src/webgpu/api/validation/resource_usages/textureUsageInPassEncoder.spec.ts

```diff
@@ -984,9 +993,13 @@ g.test('unused_bindings_in_pipeline')
     if (callDrawOrDispatch) t.issueDrawOrDispatch(pass, compute);
     pass.endPass();
 
+    // Resource usage validation scope is defined by dispatch calls. If dispatch is not called,
+    // we don't need to do resource usage validation and no validation error to be reported.
+    const success = compute && !callDrawOrDispatch;
```

Comment:
Shouldn't this be:
```suggestion
    const success = compute || !callDrawOrDispatch;
```

Reply:
I think this should be a "&&", because the validation scope in render pass is the pass, regardless we call draw command. However, the validation scope for compute is each dispatch call. If we don't call dispatch, no validation scope at all (so it will success).

Reply:
Oh yes, sorry. I have a hard time with boolean expressions often.